### DISCUSSION TITLE
UDPBD for ps2

### DISCUSSION THREAD

**Discussion by binarygeek119** [retronas/discussions#9] [Ideas]
UDPBD - The next generation of loading games through LAN on a PS2!
https://www.psx-place.com/threads/testers-needed-udpbd-the-next-generation-of-loading-games-through-lan-on-a-ps2.40132/

**Comment by sairuk** [comment]
alpha and no source code?

would be cool eventually from the sounds of it though

**Reply by coldjim** [reply]
https://gitlab.com/ps2max/udpbd-server

does this help? Seems like a great solution for ps2 game loading if you can implement to retronas

**Reply by sairuk** [reply]
it may thanks, we could add support for this server but until the image manager is written the block device will need to be setup manually

**Comment by sairuk** [comment]
support for UDPBD is now added to the experimental menu in the testing branch. Apart from getting the service running on the defaults it is completely untested as I don't have time to do so atm and as such there is no documentation yet.

1. Install it
2. select your block device 
3. then enable it (it is disabled by default) 

if you want to change block devices later you select the new block device and restart the service.

if you can test it and document the process/opl builds etc please do so here

**Reply by coldjim** [reply]
Thanks for this, when i get a chance i will test it
